nautobot_device_lifecycle_mgmt/tests/test_views.py

```diff
@@ -114,6 +117,12 @@ def test_validation_report_view_with_permission(self):
             200,
         )
 
+    def test_get_object_notes(self):
+        pass
+
+    def test_list_objects_unknown_filter_no_strict_filtering(self):
+        pass
```

Comment:
This should work after the change to the base filterset class.

Reply:
So I had to add it for the Views that explicitly call `filterset` in some of the methods https://github.com/nautobot/nautobot-plugin-device-lifecycle-mgmt/blob/pre-v1.1.0-housekeeping/nautobot_device_lifecycle_mgmt/views.py#L530.

This causes multiple warnings, instead of just one, tripping the test:

```
FAIL: test_list_objects_unknown_filter_no_strict_filtering (nautobot_device_lifecycle_mgmt.tests.test_views.ValidatedSoftwareDeviceReportViewTest)
Verify that without STRICT_FILTERING, an unknown filter is ignored.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/test/utils.py", line 387, in inner
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/nautobot/utilities/testing/views.py", line 599, in test_list_objects_unknown_filter_no_strict_filtering
    self.assertEqual(
AssertionError: Lists differ: ['WAR[108 chars]vor"', 'WARNING:nautobot.utilities.filters:Dev[193 chars]or"'] != ['WAR[108 chars]vor"']

First list contains 2 additional elements.
First extra element 1:
'WARNING:nautobot.utilities.filters:DeviceSoftwareValidationResultFilterSet: Unknown filter field "ice_cream_flavor"'

  ['WARNING:nautobot.utilities.filters:DeviceSoftwareValidationResultFilterSet: '
-  'Unknown filter field "ice_cream_flavor"',
-  'WARNING:nautobot.utilities.filters:DeviceSoftwareValidationResultFilterSet: '
-  'Unknown filter field "ice_cream_flavor"',
-  'WARNING:nautobot.utilities.filters:DeviceSoftwareValidationResultFilterSet: '
   'Unknown filter field "ice_cream_flavor"']
```